packages/frontend/src/components/other-sites/OtherSites.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { OutLink } from '../OutLink'
+
+export function OtherSites() {
+  return (
+    <div
+      className="
+      mt-8 flex flex-col items-center justify-between
+      rounded-xl bg-gradient-to-r from-purple-100/40
+      via-pink-100/40
+      to-red-200/40 px-10
+      py-6 md:flex-row
+    "
+    >
+      <div className="text-center text-lg text-white md:text-left">
+        Ethereum Layer 1 is expensive. How much does it cost to use Layer 2? 💸
+      </div>
+      <OutLink
+        className="
+        mt-3 ml-0 min-w-[90px] self-center text-center
```

Comment:
```suggestion
        mt-3 ml-0 min-w-[90px]
```
Correct me if I'm wrong, but it seems to me that those classes are not necessary (we're centering this link by flex already).

Reply:
Yeah, "self-center" is superfluous, but "text-center" is needed